.github/workflows/build-images-beta.yaml

```diff
@@ -18,6 +18,17 @@ permissions:
   id-token: write
 
 jobs:
+  add-inputs-to-env:
+    if: ${{ github.event_name == "workflow_dispatch" }}
+    name: Add Workflow Dispatch Inputs to Env
+    environment: release-beta-images
+    runs-on: ubuntu-22.04
+    env:
+      INPUTS: ${{ tojson(inputs) }}
+    steps:
+      - run: |
+          echo See env for inputs
+
   build-and-push:
     timeout-minutes: 45
     name: Build and Push Images
```

Comment:
A nit, we could just print the `inputs` too, instead of setting it as env. But this should work as well.

Reply:
I had originally went this route so we didn't have to worry about any malicious input injection. For instance, if we echoed the inputs, someone could provide `' && echo "hello world"  #` as an input:

```
echo '${{ toJson(inputs) }}' ->
echo '{"SHA": "' && echo "hello world"  # ", "PR-number": ...
```

But on second thought this is probably overkill, since you have to have elevated permissions in the repository anyways to use the `workflow_dispatch` trigger. Changed.